index.js

```diff
@@ -119,6 +119,7 @@ const plugin = fp(async function (app, opts) {
 
   if (subscriptionOpts) {
     subscriber = new PubSub(emitter)
+    app.decorate('pubsub', subscriber)
```

Comment:
This should be added to the `fastifyGraphql` object like https://github.com/mcollina/fastify-gql/pull/261/files#diff-168726dbe96b3ce427e7fedce31bb0bcR230.